tests/api/test_types.py

```diff
@@ -104,3 +105,23 @@ def test_valid_human_handle(email, label):
 def test_invalid_human_handle(email, label):
     with pytest.raises(ValueError):
         HumanHandle(email, label)
+
+
+def test_sas_code():
+    assert SASCode.from_int(0x0) == SASCode("AAAA")
+    assert SASCode.from_int(0x1) == SASCode("BAAA")
+    # [...]
+    assert SASCode.from_int(0x84001) == SASCode("BASS")
+    # [...]
```

Comment:
no test on `1337` ? :trollface:

Reply:
`1337` is not valid, but `L337` is :wink: